packages/tax/src/services/tax-module-service.ts

```diff
@@ -78,4 +84,48 @@ export default class TaxModuleService<TTaxRate extends TaxRate = TaxRate>
   ) {
     return await this.taxRateService_.create(data, sharedContext)
   }
+
+  async createTaxRegions(
+    data: TaxTypes.CreateTaxRegionDTO[],
```

Comment:
nit: Allow the creation of a single tax region. Don't think it's important; would only be for consistency purposes

Reply:
Yep would add that later as well!